### PR TITLE
Remove ea/origin

### PR DESCRIPTION
**Pull request by unspec**
Remove EA/Origin from the list, due to switch to HTTPS for downloads :(

### CODE REVIEW

**Comment by mintopia** [comment]
See Issue #98.

**Comment by Lepidopterist** [comment]
Are we happy to merge today, or do we want to personally validate?

**Comment by unspec** [comment]
I'm assuming it will still be functional as SNI Proxy is already on for this CDN, so it will be passing through but not caching. On that basis there is no immediate rush to merge, unless somebody shouts that downloads are actually broken.

**Comment by mintopia** [comment]
Agree on holding off until we see any outcomes of approaches to EA.

**Comment by VibroAxe** [comment]
I would suggest if we do we back out the delete on the origin.txt and just remove it from the json

**Comment by GotenXiao** [comment]
Honestly, it might be worth merging this until it's resolved, and then revert the merge commit if they switch back to HTTP.

**Comment by Mojo-LB** [comment]
Any ideas if there is a check for SSL thumbprint check implemented in the client? If not then spoofing with self-signed certs that's trusted on LAN PCs might be an option

**Comment by Crabbey** [comment]
I don't believe we know yet if they are doing cert pinning.

Spoofing a CA is always an option for running a man-in-the-middle, but it is very much not recommended. Of course, if you have complete control over all the clients, and you own them all, it's an easy implementation.
Installing a rogue CA on machines you don't own is not a good idea - or good solution - though. It would weaken the trust in all "secure" operations the users on those machines are doing.

tl;dr: might work, but really don't.

**Comment by Mojo-LB** [comment]
Of course, that would be for PCs owned, not on a public network. A lot of areas in the world still operate network LANs and a place with 100 PCs and 30Mbps internet connection, and with Apex's popularity, I really don't want to configure file copy scripts like we are currently doing with Epic/Fortnite

**Comment by Crabbey** [comment]
Cool. Like I say, it'll probably work, but it's not generally considered a tenable solution.
Also, FWIW, EpicGames is now cacheable, following a conversation UKLans had with Epic, so you shouldn't need to MITM that :)

**Comment by Mojo-LB** [comment]
> Also, FWIW, EpicGames is now cacheable, following a conversation UKLans had with Epic, so you shouldn't need to MITM that :)

Oh thanks for the headsup! gotta start on that bit first!
